bodo/tests/test_streaming/test_window.py

```diff
@@ -1435,7 +1435,7 @@ def impl(in_df):
                         [
                             None
                             if i % 3 == 1
-                            else bodo.types.Time(nanosecond=int(1.5**i))
+                            else bodo.types.Time(microsecond=int(1.5**i))
```

Comment:
Pandas testing functions convert Arrow time to datetime.time which loses the nanosecond part. This is just a testing issue so avoiding nanoseconds in tests for now.